### DISCUSSION TITLE
Adding some extensions

### DISCUSSION THREAD

**Discussion by Legopapurida** [Neoteroi/BlackSheep#151] [Ideas]
It will be cool if we can make or use extension feature like flask
I want to provide a list of my best extensions that I use so much in flask:

1. LoginManager
2. Email
3. WTF
4. SQLAlchemy
5. Bcrypt
6. Admin
7. SocketIo
8. debugtoolbar
9. Imagetools
10. Migrate
11. Mongoengine

**Comment by myusko** [comment]
Hi @Legopapurida 

Please, check my answer here https://github.com/Neoteroi/BlackSheep/discussions/143#discussioncomment-797347

Thanks,
Mykhailo
